README.md

```diff
@@ -72,7 +72,8 @@ available at this time are `minimal`, `default`, and `complete`:
 
 * The **minimal** profile includes as few components as possible to get a
 working compiler (`rustc`, `rust-std`, and `cargo`). It's recommended to use
-this component on Windows systems if you don't use local documentation, and in
+this component on Windows systems if you don't use local documentation (the large
+number of files causes issues on that OS), and in
```

Comment:
```suggestion
number of files can cause issues with some AV systems), and in
```

Reply:
thanks for that @kinnison... can you spell it out

Reply:
Can I spell what out?

Reply:
AV, to reduce the need to guess. Let me do it and squash the commits.